extensions/podman/packages/extension/src/configuration/registry-configuration.ts

```diff
@@ -79,7 +82,95 @@ export class RegistryConfigurationImpl implements RegistryConfiguration {
     return commands.registerCommand('podman.setupRegistry', () => this.setupRegistryCommandCallback());
   }
 
-  async checkRegistryConfFileExistsInVm(): Promise<boolean> {
+  async loadDefaultUserRegistries(): Promise<void> {
+    if (env.isMac || env.isWindows) {
+      const checked = await this.checkRegistryConfFileExistsInVm(false);
+      if (!checked) {
+        return;
+      }
+    }
+
+    const defaultRegistries: RegistryConfigurationEntry[] = [];
+    const userDefaultRegistries = configuration.getConfiguration('registries').get('defaults', []) as (
+      | DefaultRegistry
+      | DefaultRegistryMirror
```

Comment:
question: why is there a cast here, and no cast in image-registry when getting `getConfiguration('registries').get('defaults', [])`?